packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.controller.js

```diff
@@ -0,0 +1,84 @@
+export default class VpsScheduleBackupCtrl {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, Poller, VpsService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.Poller = Poller;
+    this.VpsService = VpsService;
+    this.model = {
+      rescheduleTimepicker: null,
+    };
+  }
+
+  $onInit() {
+    this.model.rescheduleTimepicker = this.automatedBackupOption.schedule;
+    this.isLoading = false;
+  }
+
+  /**
+   * Wait the triggered task
+   * @param taskId {Number}: id of running task
+   * @returns {Promise}: task promise
+   */
+  waitRescheduleAutomatedBackupTask(taskId) {
+    const endPointUrl = `/vps/${this.serviceName}/tasks/${taskId}`;
+    return this.Poller.poll(endPointUrl, null, {
+      interval: 1000,
+      successRule(task) {
+        return task.state === 'done';
+      },
+      namespace: 'dedicated.vps.veeam.reschedule-automated-backup',
+      notifyOnError: false,
+    });
+  }
+
+  /**
+   * Triggered on customer click on cross modal button
+   * @returns {Promise}
+   */
+  onModalCanceled() {
+    return this.goBack();
```

Comment:
Do we need a specific function for this ? Can't we juste use `$ctrl.goBack` in template ?

Reply:
Updated.